testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/remote/security/EJBUtil.java

```diff
@@ -52,19 +52,19 @@
      * @throws NamingException
      */
     @SuppressWarnings("unchecked")
-    public static <T> T lookupEJB(Class<? extends T> beanImplClass, Class<T> remoteInterface) throws NamingException {
+    public static <T> T lookupEJB(Class<? extends T> beanImplClass, Class<T> remoteInterface, String host) throws NamingException {
         final Hashtable<String, String> jndiProperties = new Hashtable<String, String>();
-        jndiProperties.put(Context.URL_PKG_PREFIXES, "org.jboss.ejb.client.naming");
-        //        jndiProperties.put("jboss.naming.client.ejb.context", "true");
+        jndiProperties.put(Context.INITIAL_CONTEXT_FACTORY, "org.wildfly.naming.client.WildFlyInitialContextFactory");
+        jndiProperties.put(Context.PROVIDER_URL, "remote+http://"+host+":"+8080);
         final Context context = new InitialContext(jndiProperties);
 
-        return (T) context.lookup("ejb:/" + APPLICATION_NAME + "/" + beanImplClass.getSimpleName() + "!"
+        return (T) context.lookup("ejb:/" + APPLICATION_NAME + "//" + beanImplClass.getSimpleName() + "!"
```

Comment:
Adding the extra slash here to get this test to pass is okay for now. However, it would be good to look into why the distinct name is no longer being parsed properly without it.